backend/src/routes.js

```diff
@@ -112,4 +112,17 @@ router.get('/ngos', async (req, res) => {
     };
 });
 
+
+// endpoint: Update an NGO
+router.put("/ngos/:id", async (req, res) => {
+    const ngo_id = req.params.id;
+    const ngo_data = req.body;
+    try {
+        await NGO.findByIdAndUpdate(ngo_id, { $set: ngo_data });
```

Comment:
Lets handle the result of the findByIdAndUpdate method before we send back the response. You can find the response object of this method in the documentation, where you can find some properties that indicate whether the update was successful or not. Lets return 200 only in this case and 400 if the update does not actually happen. (But keep the catch clause that causes the 404 response).
When you do, please update the PUT User endpoint to do the same handling as well. Thanks!